### ISSUE TITLE
!!! DON'T REPORT ISSUES HERE !!!

### ISSUE THREAD

**Issue by kyrios123** [solus-project/brisk-menu#120]
This repository is deprecated. Instead use the [getsolus](https://github.com/getsolus/brisk-menu) fork.

Thanks!

**Comment by allanlaal** [comment]
could someone @solus-project please mark this repo as Archived!

**Comment by kyrios123** [comment]
@allanlaal nope. If we could it would have been done a long time ago already.
